js/models/contact_model.js

```diff
@@ -436,7 +436,7 @@ angular.module('contactsApp')
 		} else {
 			angular.extend(this.props, {
 				version: [{value: '3.0'}],
-				fn: [{value: t('contacts', 'New contact')}]
+				fn: [{value: t('contacts', '')}]
```

Comment:
The `t()` function translates a string to the local language. We don't need to translate `''`. So just replace `t('contacts', '')` with `''`.